### ISSUE TITLE
Trouble with Reverse Proxy and websockets

### ISSUE THREAD

**Issue by SISheogorath** [hackmdio/hackmd#198]
When I use the latest docker image for 0.4.5 my current reverse Proxy setup isn't working anymore. The websocket requests are timing out. But it works fine with 0.4.4. 

I start the docker container using:

```bash
docker run --name hackmd -e HMD_DOMAIN=pad.shivering-isles.com -e HMD_PROTOCOL_USESSL=1 -e HMD_URL_ADDPORT=0 -e POSTGRES_USER=hackmd -e POSTGRES_PASSWORD=somepassword-e HMD_USECDN=0 --link hackmd-database:hackmdPostgres -p 127.0.0.1:3000:3000 --rm hackmdio/hackmd:latest
```

My current apache reverse proxy config looks like this:
```apacheconf
<VirtualHost *:443>
        #...(hostname stuff)
        ProxyPreserveHost On
        ProxyRequests off
        ProxyPass /.wellknown !
        RewriteEngine On
        RewriteCond %{HTTP:Connection} Upgrade [NC]
        RewriteCond %{HTTP:Upgrade} websocket [NC]
        RewriteRule /(.*) ws://127.0.0.1:3000/$1 [P,L]

        ProxyPass / http://127.0.0.1:3000/
        ProxyPassReverse / http://127.0.0.1:3000
        ProxyVia On
        SSLEngine On
        #....(sslstuff)
</VirtualHost>

```

Right now I switched back to the setup of https://github.com/hackmdio/docker-hackmd/pull/2

Any ideas why it fails for the latest official container?

**Comment by jackycute** [comment]
Hey @SISheogorath, it's not ready yet for 0.4.5 release.
Please wait for few hours, we're working on it.

**Comment by SISheogorath** [comment]
Oh okay, that would explain it ^^

**Comment by jackycute** [comment]
By the way, you should use `true` or `false` instead of `1` or `0` for your environment variables.

**Comment by SISheogorath** [comment]
Okay, I changed it ^^

It may helps you to finalize:

The error is that the "status" is switching between "Offline" and "Connected" but never really get a user.

`docker logs` show use the following, beautiful lines:

```
2016-10-11T07:40:44.272Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:40:51.303Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:40:58.512Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:06.660Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:15.647Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:24.643Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:34.677Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:43.646Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:41:52.698Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:01.646Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:09.637Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:17.639Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:26.641Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:36.658Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:44.649Z - info: deserializeUser: 364d2faa-...
2016-10-11T07:42:54.646Z - info: deserializeUser: 364d2faa-...
```

**Comment by jackycute** [comment]
Yeah, because there is an package missing files, will fix that soon.

**Comment by jackycute** [comment]
Fixed in 4188d8f942867c27121d523a47899d726196213d
